src/OpenTelemetry.SemanticConventions/scripts/templates/registry/weaver.yaml

```diff
@@ -0,0 +1,48 @@
+params:
+
+  # excluded attributes will not be generated
+  # this behavior is fully controlled by jinja templates
+  excluded_attributes: ["messaging.client_id", "otel.library.name", "otel.library.version"]
```

Comment:
Any reason why you didn't include otel.library.name and otel.library.version? They are deprecated but has to be propagated according to https://github.com/open-telemetry/opentelemetry-specification/blob/v1.36.0/specification/common/mapping-to-non-otlp.md#instrumentationscope.

Reply:
Because there's no `brief` for them so there's no `summary` and the build fail due to not having docs for public members. I fixed it in semconv so once that is released, bumping the version and removing them from here will bring them back.